### PR TITLE
Added python script to check for handler pairs

### PR DESCRIPTION
**Pull request by acron0**

### CODE REVIEW

File: scripts/check-failures.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+# grep
+proc = subprocess.Popen(["grep",
+                               "-orE",
+                               "\[:[a-z-]+\s:(success|failure)\]",
+                                SRC_DIR],
+                               stdout=subprocess.PIPE)
+
+# regex
+line_regex = re.compile(ur'^(.*.cljs):\[(:[a-z-]+)\s:(success|failure)')
+results = []
```
Comment (Eleonore9) [inline scripts/check-failures.py]: I think you could build `results` using list comprehension. Just fyi as I don't think it changes much in that case.
Something like:

```
results = [re.search (line_regex , line.rstrip().replace(SRC_DIR, '')).groups () 
                  for line in iter(proc.stdout.readline,'')]
```

Btw nice use of iterators!

File: scripts/check-failures.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+for line in iter(proc.stdout.readline,''):
+    results.append(re.search (line_regex , line.rstrip().replace(SRC_DIR, '')).groups ())
+
+groups = {}
+for entry in results:
+    if not groups.has_key(entry[0]):
```
Comment (Eleonore9) [inline scripts/check-failures.py:24]: This `has_key()` function is nice! Are you using Python 3?? -> Fancy

Comment (acron0) [inline scripts/check-failures.py:24]: No, this is Python 2.7.6 :snail: !

File: scripts/check-failures.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        groups[entry[0]] = []
+    groups[entry[0]].append(entry[1:])
+
+pairs = {}
+for entry in groups:
```
Comment (Eleonore9) [inline scripts/check-failures.py]: To access keys and values of the "groups" dict you can use `dict.iteritems()`
Something like

```
for k, v in groups.iteritems():
    for handle, status in v:
```

**Comment by Eleonore9** [comment]
I think you don't need to worry about complying to PEP8 for one Python script in a whole repo.
I'm used to having the `if __name__ == "__main__":` though. I'm not sure if it's the best way to go but it makes it easy to test and use bit of a script...

**Comment by Eleonore9** [comment]
See if you need the "if **name** ...". Otherwise lgtm :shipit:

**Pull request merged**